test/Algebra/Graph/Test/Generic.hs

```diff
@@ -1763,8 +1763,8 @@ testBfsForestFrom (prefix, API{..}) = do
     test "isSubgraphOf (forest $ bfsForestFrom vs x) x     == True" $ \vs x ->
           isSubgraphOf (forest $ bfsForestFrom vs x) x     == True
 
-    test "bfsForestFrom (vertexList x) x                   == bfsForest x" $ \x ->
-          bfsForestFrom (vertexList x) x                   == bfsForest x
+--    test "bfsForestFrom (vertexList x) x                   == bfsForest x" $ \x ->
+--          bfsForestFrom (vertexList x) x                   == bfsForest x
 
```

Comment:
This no longer holds with new semantics

Reply:
I think plain `bfsForest` doesn't make sense in the new semantics... so perhaps, we should just drop it and rename `bfsForestFrom` to `bfsForest`?

Reply:
Yeah, I think that makes sense. In fact, fgl doesn't export `bfsForest` or an equivalent either

Reply:
👍